add-on/src/lib/options.js

```diff
@@ -31,7 +33,11 @@ export const optionDefaults = Object.freeze({
   importDir: '/ipfs-companion-imports/%Y-%M-%D_%h%m%s/',
   useLatestWebUI: false,
   dismissedUpdate: null,
-  openViaWebUI: true
+  openViaWebUI: true,
+  telemetryGroupMinimal: true,
+  telemetryGroupMarketing: false,
+  telemetryGroupPerformance: false,
+  telemetryGroupTracking: false
```

Comment:
- rename `telemetryGroupMinimal` to `telemetryFeatures`
- remove `telemetryGroupMarketing` and `telemetryGroupTracking` (we are not doing these, afaik don't plan to do it, so no reason to trigger anyone with this language)